app/jobs/invoices/prepaid_credit_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Invoices
+  class PrepaidCreditJob < ApplicationJob
```

Comment:
Looks good!

My only comment would be to suggest to move this business logic into a dedicated service with a name clearer than the job name. Right now, it's not easy to guess what the job is doing by reading its name.

Reply:
seems reasonable. I also wasn't 100% happy with the naming but in the same job we handle `WalletTransaction` and `Wallet` so I wasn't sure how to name it...
Moved the logic into new service and I have named it `Wallets::ApplyPaidCreditsService`